cmake/OpenCVFindLibsPerf.cmake

```diff
@@ -51,7 +51,12 @@ endif(WITH_CUDA)
 
 # --- Eigen ---
 if(WITH_EIGEN AND NOT HAVE_EIGEN)
-  find_package(Eigen3 QUIET)
+  if(NOT OPENCV_SKIP_EIGEN_FIND_PACKAGE_CONFIG)
+    find_package(Eigen3 CONFIG QUIET)  # Ceres 2.0.0 CMake scripts doesn't work with CMake's FindEigen3.cmake module (due to missing EIGEN3_VERSION_STRING)
```

Comment:
Another option might be to adjust the `FindEigen3.cmake` module to be compatible with Eigen's own cmake config modules and by (additionally) setting `EIGEN3_VERSION_STRING`.

Reply:
`FindEigen3.cmake` is not a part of OpenCV.
It is part of CMake distributions on some platforms (like Fedora 31/32, check `/usr/share/cmake/Modules/FindEigen3.cmake`).

Reply:
Oh ok, sorry, I misinterpreted it. (Still you could set `EIGEN3_VERSION_STRING` here if it is unset). But maybe your proposed solution is good enough. The only case I can think of is if someone wants to build with ceres 2.0.0, but find Eigen in a way that works with the find module, but doesn't work with eigens own config module (maybe from an Eigen source tree directly, not build or install dir).